client/src/components/rounds/Apply.tsx

```diff
@@ -179,6 +191,7 @@ function Apply() {
             {props.applicationMetadata !== undefined && (
               <Form
                 roundApplication={props.applicationMetadata}
+                showErrorModal={props.showErrorModal || false}
```

Comment:
I think `showErrorModal` can be changed to be just `applicationStatus === ApplicationStatus.Error` so that here it's directly the boolean we need

Reply:
when I do this the Done button does not work (close the modal)